library/Vanilla/Models/ThemeModel.php

```diff
@@ -405,7 +416,7 @@ public function getCurrentTheme(): ?array {
                 $current = $provider->getThemeWithAssets(self::FALLBACK_THEME_KEY);
             }
         } catch (\Exception $e) {
-            trigger_error($e->getMessage(), E_USER_ERROR);
+            trigger_error($e->getMessage(), E_USER_NOTICE);
```

Comment:
Why are we downgrading this to a notice? I figure it should be a warning at least if someone has an invalid theme configured.

Reply:
I set it to Warning